hack/remote-up-karmada.sh

```diff
@@ -38,14 +41,29 @@ then
   exit 1
 fi
 
+# Make sure go exists and the go version is a viable version.
```

Comment:
Do we use `go` in `hack/remote-up-karmada.sh`?

Reply:
Yes .
see:
```
# ./hack/remote-up-karmada.sh ~/.kube/config local
go install github.com/cloudflare/cfssl/cmd/...@v1.5.0
Generating a 2048 bit RSA private key
...........+++
........+++
writing new private key to '/root/.karmada/server-ca.key'
-----
Generating a 2048 bit RSA private key
.................+++
............................................+++
writing new private key to '/root/.karmada/front-proxy-ca.key'
-----
2022/05/23 09:35:05 [INFO] generate received request
2022/05/23 09:35:05 [INFO] received CSR
2022/05/23 09:35:05 [INFO] generating key: rsa-2048
2022/05/23 09:35:06 [INFO] encoded CSR
2022/05/23 09:35:06 [INFO] signed certificate with serial number 19299581622024272790925947613243113268521003237
2022/05/23 09:35:06 [INFO] generate received request
2022/05/23 09:35:06 [INFO] received CSR
2022/05/23 09:35:06 [INFO] generating key: rsa-2048
2022/05/23 09:35:08 [INFO] encoded CSR
2022/05/23 09:35:08 [INFO] signed certificate with serial number 365083759225180248063143554959235905759130933715
```